eras/alonzo/test-suite/test/Tests.hs

```diff
@@ -56,10 +62,18 @@ nightlyTests :: TestTree
 nightlyTests =
   testGroup
     "Alonzo tests"
-    [ alonzoPropertyTests, -- These are the full property tests
+    [ incrementalTests,
```

Comment:
can we put `incrementalTests` inside `alonzoPropertyTests`?

Reply:
Done